connector/subgraph/manifest/data/rinkeby.json

```diff
@@ -1,98 +1,16 @@
-        "startBlock": 5922866
-      },
-      {
-        "name": "DAOFactory@0.8",
-        "address": "0xad4d106b43b480faa3ef7f98464ffc27fc1faa96",
-        "startBlock": 5026118
-      },
-      {
-        "name": "DAOFactory@0.7",
-        "address": "0xfdef49fbfe37704af55636bdd4b6bc8cd19143f6",
-        "startBlock": 4221581
-      },
-      {
-        "name": "DAOFactory@0.6",
-        "address": "0x2298d27a9b847c681d2b2c2828ab9d79013f5f1d",
-        "startBlock": 3223971
-      }
-    ],
-    "OrganizationTemplates": [
-      {
-        "name": "bare-template.aragonpm.eth@1.0.0",
-        "address": "0x789e4695d4D24EBFAcbccDd951A3D4075C5ce261",
-        "startBlock": 5037492
-      },
-      {
-        "name": "company-template.aragonpm.eth@1.0.0",
-        "address": "0xA3809a525B92a8A290E5d704f9Be6B5046506A7b",
-        "startBlock": 5037534
-      },
-      {
-        "name": "company-board-template.aragonpm.eth@1.0.0",
-        "address": "0xc1e287844b3445a46B30244fc84663F8740b3b1E",
-        "startBlock": 5037527
-      },
-      {
-        "name": "dandelion-org-template.aragonpm.eth@1.0.0",
-        "address": "0x09380aeaf234052bb8bde6dceffc45d8fde8f98d",
-        "startBlock": 5925084
-      },
-      {
-        "name": "fundraising-multisig-template.aragonpm.eth@1.0.0",
-        "address": "0x5671D4815A2a5cFbF541496406460bF56dff2a01",
-        "startBlock": 5232230
-      },
-      {
-        "name": "fundraising-multisig-template.aragonpm.eth@2.0.0",
-        "address": "0x4b514B5E57E9A42ab24007F6253cD47031B5ed2f",
-        "startBlock": 5608072
-      },
-      {
-        "name": "membership-template.aragonpm.eth@1.0.0",
-        "address": "0x49C980de0bD76b64d242d3dbA042bF58942E4377",
-        "startBlock": 5037542
-      },
-      {
-        "name": "reputation-template.aragonpm.eth@1.0.0",
-        "address": "0xefe6Da5fdFEfAbe96E41aCdcb5689420cEE894Cd",
-        "startBlock": 5037548
-      }
-    ],
-    "LegacyOrganizationTemplates": [
-      {
-        "name": "bare-kit.aragonpm.eth@1.0.0",
-        "address": "0x204b52317f23AEE87e440905059d6bD0a7B0104D",
-        "startBlock": 3633430
-      },
-      {
-        "name": "democracy-kit@1.0.0",
-        "address": "0x89D31C0AFD8367960fc0c9d11225630d4944550E",
-        "startBlock": 3229645
-      },
-      {
-        "name": "democracy-kit@2.0.0",
-        "address": "0xB99DE66D3dCEEe078071D67D18FBBae1Cb44eC2B",
-        "startBlock": 4221693
-      },
-      {
-        "name": "multisig-kit.aragonpm.eth@1.0.0",
-        "address": "0x01F2e9CDAac527Ffef1B9869d4C0B33357c0A2Bc",
-        "startBlock": 3229673
-      },
-      {
-        "name": "multisig-kit.aragonpm.eth@2.0.0",
-        "address": "0xC00EAD10F0A097C71f4b7999530A1F2d9E92a8a5",
-        "startBlock": 4221706
+        "name": "testdao",
+        "address": "0xace2faa375ffc251fafc7360bc3defbc6184e5fe",
+        "startBlock": "7013510"
       }
     ],
```

Comment:
Is this intended? We can use the staging subgraph for this.

Reply:
not sure how to proceed with this since, we are not tracking the staging subgraph from the connector https://github.com/1Hive/conviction-voting-app/blob/a5652436af1ed56321afbc836720a0fb09abedbd/connector/src/thegraph/connector.ts#L10-L20, and this subgraph will track old conviction voting smart contracts and i guess that will fail